config/routes.rb

```diff
@@ -5,4 +5,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :tasks, only: %(index)
```

Comment:
What do you think if we use square brackets for arrays?
```suggestion
  resources :tasks, only: %[index]
```